### DISCUSSION TITLE
Is it possible to disable onekeys?

### DISCUSSION THREAD

**Discussion by Epaclon** [kingToolbox/WindTerm#1987] [Q&A]
I come from version 2.5.0 and have many saved sessions. Many of them change the password every few months.

I would like to know if there is a way to disable the onekeys function, or at least if there is a way to make onekeys not the default tab when logging into a saved session. 

The function seems interesting to me but I have many saved sessions from previous versions and when logging in, it is uncomfortable to have to take the mouse to select in the login window the account tab, to be able to enter the password.

**Comment by apdma** [comment]
I found, for the fast and dirty fix, you can comment settings in the wind.config
```
"terminal.commandHistory.storage": true,
//  "terminal.expect.password": "password:|password for \\S+:",
//  "terminal.expect.username": "username:|login:",
  "terminal.localModeEnabled": true,
//  "terminal.loginWizard.defaultAuthType": "OneKey",
  "terminal.mouseTracking.click": true,```

But i can't find the way to disable pop-up menu in console

**Comment by zdltvxq** [comment]
I have the same problem

**Comment by GILLIBON** [comment]
It would be great if it was possible to disable the pop-ups
